src/Tests/Microsoft.DotNet.ApiCompat.IntegrationTests/Task/ValidatePackageTargetIntegrationTests.cs

```diff
@@ -343,6 +344,28 @@ public void ValidateMissingReferencesIsOnlyLoggedWhenRunningWithReferences(bool
                 Assert.NotEmpty(log.warnings.Where(e => e.Contains("CP1003")));
         }
 
+        [RequiresMSBuildVersionFact("17.0.0.32901", Skip = "https://github.com/dotnet/sdk/issues/23533")]
+        public void ValidateReferencesAreRespectedForPlatformSpecificTFMs()
```

Comment:
Is this test reproing the issue reported by the customer? If not, could you add one that closely resembles the customer scenario?

Reply:
Yes, the test reproduces the customer reported issue by using a windows platform TFM.